lemur/plugins/lemur_digicert/plugin.py

```diff
@@ -327,6 +347,13 @@ def __init__(self, *args, **kwargs):
             }
         )
 
+        # max_retries applies only to failed DNS lookups, socket connections and connection timeouts,
+        # never to requests where data has made it to the server.
+        # we Retry we also covers HTTP status code 400, 406, 500, 502, 503, 504
+        retry_strategy = Retry(total=3, backoff_factor=0.1, status_forcelist=[400, 406, 500, 502, 503, 504])
```

Comment:
Can you share more context behind retry specifically for 400? Since 4xx is a bad request (exception of 406, where we are trying to fix the header), will retry of same request help?

Reply:
I had noticed the DigiCert 406 being accompanied with a 400, hence the thinking there. But we can remove it for now, and see how the state recovery after 406 helps.